src/main/java/org/jenkinsci/plugins/durabletask/Handler.java

```diff
@@ -34,7 +34,7 @@
 
 /**
  * A remote handler which may be sent to an agent and handle process output and results.
- * If it needs to communicate with the master, you may use {@link VirtualChannel#export}.
+ * If it needs to communicate with the Controller, you may use {@link VirtualChannel#export}.
```

Comment:
```suggestion
 * If it needs to communicate with the controller, you may use {@link VirtualChannel#export}.
```
I assume the same here